spec.bs

```diff
@@ -8637,6 +8679,20 @@ A <dfn>direct from seller signals</dfn> is a [=struct=] with the following [=str
     JSON data passed to corresponding buyer's [=script runner=].
 </dl>
 
+<h3 id=additional-bids-structs-section>Additional bids</h3>
+
+A <dfn>signed additional bid with metadata</dfn> is a [=struct=] with the following
+[=struct/items=]:
+
+<dl dfn-for="signed additional bid with metadata">
+  : <dfn>signed additional bid</dfn>
+  :: A [=string=] for an encoded additional bid.
+  : <dfn>seller nonce</dfn>
+  :: Null or a [=string=]. If present, represents the randomly-generated seller nonce returned in
+    the [:Ad-Auction-Additional-Bid:] response header. This must be 36 characters, and should be the

```

Comment:
This language is perfect, with the clear distinction between the 'must' of 36 characters and the 'should' of canonical string representaiton of a UUIDv4. Thanks, Caleb!